tests/pre_merge/datasets/test_synthetic_data.py

```diff
@@ -30,24 +30,53 @@ def get_folder_dataset():
 
 
 @pytest.fixture(autouse=True)
-def make_synthetic_dataset():
+def make_perlin_synthetic_dataset():
     """Create synthetic anomaly dataset from folder dataset."""
 
     def make():
         folder_dataset = get_folder_dataset()
-        synthetic_dataset = SyntheticAnomalyDataset.from_dataset(folder_dataset)
+        synthetic_dataset = SyntheticAnomalyDataset.from_dataset(folder_dataset, TestSyntheticType.PERLIN)
         return synthetic_dataset
 
     return make
 
 
 @pytest.fixture(autouse=True)
-def synthetic_dataset_from_samples():
+def make_perlinroi_synthetic_dataset():
```

Comment:
Since both the methods are very similar, we can use `mark.parameterize` https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions

Reply:
Noted, we will work on the changes. Thanks again Ashwin